docs/source/kfp.components.structures.rst

```diff
@@ -0,0 +1,21 @@
+kfp.components.structures package
+=================================
+
+
+.. automodule does not see anything here
+
+    .. automodule:: kfp.components.structures
+        :members:
+        :undoc-members:
+        :show-inheritance:
+
+.. Added manually
+
+.. automodule:: kfp.components._structures
```

Comment:
Could we not display the _structures for now since this is not the user interface?
If users need instructions to write the yaml, it should not be here.

Reply:
Let's make it the user interface then. WDYT?

I remember some people talked about using python to generate YAML.

Reply:
Removed it for now.